overlord/managers_test.go

```diff
@@ -211,7 +211,15 @@ func (ms *mgrsSuite) TearDownTest(c *C) {
 var settleTimeout = 15 * time.Second
 
 func makeTestSnap(c *C, snapYamlContent string) string {
-	return snaptest.MakeTestSnapWithFiles(c, snapYamlContent, nil)
+	info, err := snap.InfoFromSnapYaml([]byte(snapYamlContent))
+	c.Assert(err, IsNil)
+
+	var files [][]string
+	for _, app := range info.Apps {
+		files = append(files, []string{app.Command, ""})
```

Comment:
Pardon my ignorance, but why an extra empty string (`""`) here?

Reply:
the last argument of `MakeTestSnapWithFiles` is a `[][]string`, with the second string's first element being the filename, and the second being its contents